### ISSUE TITLE
Dosen't work in multiplayer

### ISSUE THREAD

**Issue by Froggjarl** [MCModderAnchor/TACZ#111]
Me and my friend are playing a custom mod pack with this mod in it. Every time I shoot, he gets disconnected. He also can't shoot. he can do everything else with the gun, he just can't shoot it. Any suggestions?

**Comment by 286799714** [comment]
please provide game log

**Comment by Froggjarl** [comment]
Sorry, but the only logs I could find were in the logs folder, all of which were too long to send. I have included the mod list if that is any help.

[Starter Kit (by Serilum
[From The Fog (by LunarEclipseStudios)
[Radium Reforged (by Asek3)
[Configured (by MrCrayfish)
[Fresh Animations (by FreshLX)
[Just Enough Resources (JER) (by way2muchnoise)
[Just Enough Items (JEI) (by mezz)
[Marbled's API [MAPI] (by MarbledNull)
[Timeless and Classics Zero (by nekocrane)
[Collective (by Serilum)
[Marbled's Arsenal [MA] (by MarbledNull)
[Mouse Tweaks (by YaLTeR)
[Embeddium (by embeddedt)
[GeckoLib (by Gecko)
[Embeddium/Rubidium Extras (by Txni)
[The Man From The Fog (by M_Productions)
[Marbled's Villager Hats [MVH] (by MarbledNull)
[Oculus (by Asek3)
[Essential Mod (by SparkUniverse_)
[Default Dark Mode (by nebulr)
[GoatMan (by Llamii_Llama)
[Complementary Shaders - Reimagined (by EminGT)
[The Knocker (M_Productions)

At the time of writting this I did update TACZ, which might have done something.
